vllm/distributed/kv_transfer/kv_lookup_buffer/base.py

```diff
@@ -0,0 +1,25 @@
+from abc import ABC, abstractmethod
+from typing import List, Optional
+
+import torch
+
+
+class KVLookupBufferBase(ABC):
```

Comment:
Can you add docstring to describe what is this for? Also docstring each abstract method.

Reply:
Done :smile: